src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/ImageController.php

```diff
@@ -60,7 +60,8 @@ class ImageController extends FrameworkBundleAdminController
     public function getImagesAction(int $productId): JsonResponse
     {
         /** @var ProductImage[] $images */
-        $images = $this->getQueryBus()->handle(new GetProductImages($productId));
+        $images = $this->getQueryBus()->handle(
```

Comment:
Not really sure it was worth a blank line but why not 😅

Reply:
It's cleary not necessary, it's just an involuntary side effect due to doing and undoing things ! 😂